syntax = "proto2";

package tenon.vss.protobuf;

message VssMessage {
	optional uint64 random_hash = 1;
	optional uint64 random = 2;
    optional bytes pubkey = 3;
	optional bytes sign_ch = 4;
	optional bytes sign_res = 5;
	optional uint64 tm_height = 6;
	optional uint64 elect_height = 7;
	optional int32 type = 8;
}